### PR TITLE
Clarify that it's safe to reuse an FFI scope.

### PR DESCRIPTION
**Pull request by Crell**
I have been told this is true, at least.  My C knowledge is too weak to verify.

### CODE REVIEW

File: reference/ffi/ffi/scope.xml [hunk 1]
```diff
@@ -16,6 +16,10 @@
   <para>
    Instantiates an FFI object with C declarations parsed during preloading.
   </para>
+  <para>
+   The <code>scope</code> method is safe to call multiple times for the same scope. Multiple references to the
```
Comment (salathe) [inline reference/ffi/ffi/scope.xml]: Correct markup for`<code>scope</code>` would be `<methodname>FFI:scope</methodname>`.  Alternatively, "*This method is safe to call…*".

Comment (Crell) [inline reference/ffi/ffi/scope.xml]:
```suggestion
   The <methodname>FFI::scope</methodname method is safe to call multiple times for the same scope. Multiple references to the
```

File: reference/ffi/ffi/scope.xml [hunk 2]
```diff
@@ -16,6 +16,10 @@
   <para>
    Instantiates an FFI object with C declarations parsed during preloading.
   </para>
+  <para>
+   The <methodname>FFI::scope</methodname method is safe to call multiple times for the same scope. Multiple references to the
```
Comment (salathe) [inline reference/ffi/ffi/scope.xml]: Typo: missing final `>` from the closing methodname tag.

Comment (Crell) [inline reference/ffi/ffi/scope.xml]:
```suggestion
   The <methodname>FFI::scope</methodname> method is safe to call multiple times for the same scope. Multiple references to the
```

Comment (Crell) [reply reference/ffi/ffi/scope.xml]: Damnit, Crell!  Fixed.

**Comment by salathe** [comment]
@php-pulls merge

**Comment by salathe** [comment]
Thanks @crell :+1:

**Pull request closed without merging**